projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cds/profile-tag.e2e-spec.ts

```diff
@@ -0,0 +1,44 @@
+import { clickAllowAllFromBanner } from '../../../helpers/anonymous-consents';
+import { Navigation } from '../../../helpers/navigation';
+import { CdsHelper } from '../../../helpers/vendor/cds/cds';
+import * as MercahndisingCarousel from '../../../helpers/vendor/cds/merchandising-carousel';
+import { ProfileTagHelper } from '../../../helpers/vendor/cds/profile-tag';
+describe('Profile-tag component', () => {
+  beforeEach(() => {
+    cy.server();
+    CdsHelper.setUpMocks();
+  });
+  it('Should wait for a user to accept consent before sending an event', () => {
+    Navigation.visitHomePage();
+    cy.wait(1000);
+    cy.wait(1).then(() => {
```

Comment:
same as above

Reply:
This was the only way I could figure out how to make the expect statement fire in order in cypress, other wise it will fire too early. I.E. the waits happen one after another. Is there another way?

Reply:
if we wait for the cms page `homepage`, will it be fine or is it too early?

Reply:
for the wait(1), all that matters here is that there is a wait, so waiting for the homepage would work as well.
For the wait(1000), I wanted to assert that no requests are sent until we click allow all on the consent banner. I guess this is a bit against the cypress philosophy, as they do not seem to approve of negative tests, but for us I would say its a valid use case. I.E. if we sent an event without someone accepting consent, then this would be a bad bug as it wouldn't be GDPR compliant..... let me know if this is ok for you guys to have this test

Reply:
I'll have to mention this in the standup, will get back to you 8 hours from now (this will be my first question to the team)